packages/common/src/types/api/comments.ts

```diff
@@ -7,7 +7,9 @@ export type ApiCreateComment = {
 
 export type ApiGetComments = {
     status: 'ok';
-    results: ApiCommentsResults;
+    results: {
+        [key: string]: Comment[];
```

Comment:
we could use `dashboardTileUuid` instead of `key`

Reply:
done!